pkg/cluster/describe.go

```diff
@@ -240,8 +258,6 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
 		cluster.CCS().Enabled(),
 		cluster.Hypershift().Enabled(),
 		cluster.AWS().SubnetIDs(),
```

Comment:
optional : AWS().SubnetIDs , probably dont need this for GCP ?

Reply:
Included